### PR TITLE
[primTorch] Rename is_finite->isfinite

### PR DESCRIPTION
**Pull request by zasdfgbnm**
`isfinite` sounds like a better name, because PyTorch, C++, numpy all have this name instead of `is_finite`

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
Is there some unit test I need to pay attention to?

**Comment by mruberry** [comment]
> Is there some unit test I need to pay attention to?

No this is perfect, but because of https://github.com/pytorch/pytorch/issues/78058 this has to add an entry like these:

https://github.com/pytorch/pytorch/blob/3b70a7c294b3a56494912894377dccb70d9b3505/test/forward_backward_compatibility/check_forward_backward_compatibility.py#L146

I'm going to update that test (or this PR can update it, both good) to stop checking BC/FC for ops in the "prims" namespace.

**Comment by zasdfgbnm** [comment]
I can try update the test, it should not be hard.

**Comment by zasdfgbnm** [comment]
Also fixes https://github.com/pytorch/pytorch/issues/78058

**Comment by zasdfgbnm** [comment]
backwards_compat is passing now

**Review by ngimel — APPROVED** [review]

File: test/forward_backward_compatibility/check_forward_backward_compatibility.py [hunk 1]
```diff
@@ -139,11 +139,7 @@
     ("aten::_weight_norm_cuda_interface", datetime.date(9999, 1, 1)),
     ("aten::_weight_norm_cuda_interface_backward", datetime.date(9999, 1, 1)),
     # TODO: FIXME: prims shouldn't be checked
-    ("prims::any", datetime.date(9999, 1, 1)),
-    ("prims::all", datetime.date(9999, 1, 1)),
-    ("prims::empty_like", datetime.date(9999, 1, 1)),
-    ("prims::select", datetime.date(9999, 1, 1)),
-    ("prims::concatenate", datetime.date(9999, 1, 1)),
+    ("prims::.*", datetime.date(9999, 1, 1)),
```
Comment (mruberry) [inline test/forward_backward_compatibility/check_forward_backward_compatibility.py:142]: Neat

**Review by mruberry — APPROVED** [review]
Cool!

**Comment by IvanYashchuk** [comment]
Similarly `is_infinite` should be renamed to `isinf` to match PyTorch, NumPy, and C++.

**Comment by zasdfgbnm** [comment]
Yes, but it's better to remove is_infinite from prims, because it can be composed from infinite and isnan

**Pull request closed without merging**